lib/node_modules/@stdlib/math/base/special/rad2deg/package.json

```
@@ -63,5 +63,84 @@
     "degrees",
     "angle",
     "convert"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "rad2deg",
+      "alias": "rad2deg",
+      "pkg_desc": "convert an angle from radians to degrees",
+      "desc": "converts an angle from radians to degrees",
+      "short_desc": "",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "double",
+            "dtype": "float64"
+          },
+          "domain": [
+            {
+              "min": "-infinity",
+              "max": "infinity"
+            }
+          ],
+          "rand": {
+            "prng": "random/base/uniform",
+            "parameters": [
+              -10,
+              10
+            ]
+          },
+          "example_values": [
+            -12.566370614359172,
+            -6.283185307179586,
+            -3.141592653589793,
+            -1.5707963267948966,
+            -0.7853981633974483,
+            -0.5,
+            -0.25,
+            -0.1,
+            0,
+            0.1,
+            0.25,
+            0.5,
+            0.7853981633974483,
+            1,
+            1.5707963267948966,
+            2,
+            3.141592653589793,
+            4,
+            6.283185307179586,
+            12.566370614359172
+          ]
+        }
+      ],
+      "output_policy": "same",
+      "returns": {
+        "desc": "function value",
+        "type": {
+          "javascript": "number",
+          "jsdoc": "number",
+          "c": "double",
+          "dtype": "float64"
+        }
+      },
+      "keywords": [
+        "rad2deg",
+        "radians",
+        "degrees",
+        "angle",
+        "convert",
+        "trigonometry",
+        "geometry"
+      ],
+      "extra_keywords": [
+        "to-degrees"
+      ]
```

```suggestion
      "extra_keywords": []
```

You seem to misunderstand the intent of the "extra" keywords.